feature/gribi/ate_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -95,13 +97,32 @@ func shouldSkip(tt *compliance.TestSpec) string {
 		return "This RequiresNonDefaultNINHG test is skipped by --skip_non_default_ni_nhg"
 	case *skipIdempotentDelete && tt.In.RequiresIdempotentDelete:
 		return "This RequiresIdempotentDelete test is skipped by --skip_idempotent_delete"
+	case *skipMPLS && tt.In.RequiresMPLS:
+		return "This RequiresMPLS test is skipped by --skip_mpls"
 	}
 	return moreSkipReasons[tt.In.ShortName]
 }
 
+func syncElectionID(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	clientA := &gribi.Client{
+		DUT:         dut,
+		Persistence: true,
```

Comment:
If your goal of setting `PERSISTENT` mode is to make sure most current implementations can support the negotiation, then probably you want to set the redundancy mode too, right? (to `SINGLE_PRIMARY`).

Reply:
yes, the start function does this by default: https://github.com/openconfig/featureprofiles/blob/4ac3c512ce9f4974dac9892faf380d0592e3ffad/internal/gribi/gribi.go#L118